Cargo.toml

```diff
@@ -35,6 +35,9 @@ nix = { version = "0.29.0", features = ["fs", "signal", "net"] }
 libc = "0.2.168"
 arc-swap = "1.7.1"
 flume = "0.11.1"
+netavark = "1.13.1"
```

Comment:
We cannot do this here. This dependency would make it very hard for us to cut new releases.